db/migrate/20230227152546_add_name_to_users.rb

```diff
@@ -1,6 +1,5 @@
 class AddNameToUsers < ActiveRecord::Migration[7.0]
   def change
     add_column :users, :name, :string
-    has_many :offers, dependent: :destroy
```

Comment:
I noticed this is a model code, so I moved to the model.

Reply:
Also can you clarify what you mean about moving the model?

Reply:
The file that I renamed was `20230227150347_add_img_url_to_offer.rb` = singular
and the class in this file `AddImgUrlToOffers`

This is the error happening in master for me.
<img width="535" alt="image" src="https://user-images.githubusercontent.com/121885895/221645007-6b9e6d01-3240-4b00-a6d3-f570158572f3.png"> 
`

Renaming the file fixes the error.